### PR TITLE
Added youtube videos to front page

### PR DESCRIPTION
**Pull request by mangelajo**
Updated patch [1] from Tuomas with a couple of changes for discussion: 

Co-Authored-By: Tuomas Kuosmanen <tuomas.kuosmanen@gmail.com>

[1] https://github.com/submariner-io/submariner-website/pull/150

### CODE REVIEW

File: src/content/_index.en.md [hunk 1]
```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
 * [subctl](./deployment/), a friendly deployment tool 
 * Support for interconnecting clusters with [overlapping CIDRs](./architecture/globalnet/)
 
+<style>
+.mygrid {  
+    display: grid;
+    grid-gap: 12px;  
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    grid-template-rows: repeat(2, 100px);  
+    padding-bottom: 2em;
```
Comment (nyechiel) [inline src/content/_index.en.md:28]: I think we should add a bit more space. With padding-bottom set to 2cm it looks better to me (and also uncovers the h5 titles).

File: src/content/_index.en.md [hunk 2]
```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
...
+    text-align: center;
+}
+
+</style>
+
+### Examples
+<div class="mygrid">
+  <div>
+    <div> {{< youtube fMhZRNn0fxQ >}}</div>
+    <h5> Connecting containers accross clusters</h5>
```
Comment (nyechiel) [inline src/content/_index.en.md:40]: "Connecting Pods and Services across Clusters"

File: src/content/_index.en.md [hunk 3]
```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
...
+    grid-template-rows: repeat(2, 100px);  
+    padding-bottom: 2em;
+}
+.mygrid h5 {
+    text-align: center;
+}
+
+</style>
+
+### Examples
```
Comment (nyechiel) [inline src/content/_index.en.md:36]: "Demos" seems a better title to me. I think we should also use ## and not ### here, to keep the title hierarchy of this page.

File: src/content/_index.en.md (hunk 1 above)
Comment (mangelajo) [reply src/content/_index.en.md:28]: I'm not sure about what 2cm means in css :-) em I think is 16px, cm may be depends on dpi? :) I remember there were some of those which were not safe to use.. but that's all I know , can I try with more px or ems? probably Tuomas can better know about this.

File: src/content/_index.en.md (hunk 2 above)
Comment (mangelajo) [reply src/content/_index.en.md:40]: right I didn't check that :)

File: src/content/_index.en.md (hunk 3 above)
Comment (mangelajo) [reply src/content/_index.en.md:36]: :+1:

File: src/content/_index.en.md (hunk 1 above)
Comment (nyechiel) [reply src/content/_index.en.md:28]: Yeah.. wasn't sure either :)
https://www.w3schools.com/cssref/pr_padding-bottom.asp

5em is the equivalent of 2cm and also looks good to me.

Comment (mangelajo) [inline src/content/_index.en.md:28]:
```suggestion
    padding-bottom: 5em;
```

File: src/content/_index.en.md (hunk 2 above)
Comment (mangelajo) [inline src/content/_index.en.md:40]:
```suggestion
    <h5> Connecting Pods and Services across Clusters</h5>
```

File: src/content/_index.en.md [hunk 4]
```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
...
+  <div>
+    <div> {{< youtube tXsemQPNhyQ >}}</div>
+    <h5> Cross-cluster Service Discovery</h5>
+  </div>
+</div>
 
 {{% notice tip %}}
 Check the [Quickstart guide](./quickstart/) section for deployment instructions.
 {{% /notice %}}
+
```
Comment (mangelajo) [inline src/content/_index.en.md:55]:
```suggestion
```

File: src/content/_index.en.md (hunk 2 above)
Comment (mangelajo) [reply src/content/_index.en.md:40]:
```suggestion
    <h5> Connecting Pods and Services across Clusters</h5>
```

File: src/content/_index.en.md (hunk 3 above)
Comment (mangelajo) [reply src/content/_index.en.md:36]:
```suggestion
### Demos
```

**Comment by mangelajo** [comment]
@tigert I'm merging your patch with some additions on top ^

File: src/content/_index.en.md [hunk 5]
```diff
@@ -19,6 +19,35 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
...
+    grid-template-rows: repeat(2, 100px);  
+    padding-bottom: 5em;
+}
+.mygrid h5 {
+    text-align: center;
+}
+
+</style>
+
+### Demos
```
Comment (mangelajo) [inline src/content/_index.en.md:36]:
```suggestion
## Demos
```

**Pull request merged**